config/locales/devise_invitable.ptbr.yml

```diff
@@ -17,7 +17,7 @@ ptbr:
         submit_button: "Defina sua senha"
     mailer:
       invitation_instructions:
-        subject: "Você foi convidado para se juntar ao if me!"
+        subject: "{%name} convidou você para participar do if-me!"
```

Comment:
The copy here should be `if me` instead of `if-me`. I know this is a source of constant confusion, especially given our domain name lol.

Reply:
Sounds like a need for branding guidelines